### DISCUSSION TITLE
Kartu debit online dari bank mana saja yang bisa digunakan untuk membuat akun di AWS?

### DISCUSSION THREAD

**Discussion by rioastamal** [awsugid/forum#16] [General]
Jika teman-teman memiliki sebuah kartu kredit maka membuat akun AWS harusnya tidak masalah. Namun masih banyak yang tidak memiliki kartu kredit sehingga harus menggunakan kartu lain misal kartu debit online/digital card dari bank konvensional atau bank digital.

Tidak semua kartu debit online ini bisa digunakan dan kadang malah tertolak karena berbagai alasan. Thread diskusi ini untuk membagikan pengalaman bagi yang sukses menggunakan kartu debit online dari bank konvensional atau bank digital mana saja.

Thread ini merupakan salinan dari pertanyaan awal yang saya lontarkan di group Telegram AWS Indonesia https://t.me/AWSUserGroupID.

Pertanyaan original:

> Izin tanya nih, maaf msh awam di cloud computing, klu registrasi akunnya selalu bermasalah di bagian verifikasi kartu kredit/ debit ada solusi lain nggak utk bisa akses free. Terima kasih bantuannya

**Comment by rioastamal** [comment]
Berikut ini adalah kompilasi jawaban yang saya himpun dari group Telegram:

>Pake cc atau kartu jenius

---

>Bisa pake kartu debit dari blu bca
---
>Klu nggak punya kartu jenius, bca msh bisa nggak ? Maksudnya tanpa kartu perbankan msh bisa daftar kah ?? Klu pake maestro selalu error
---
>saya pake DBS bank bisa
---
>Sayangnya untuk pembayaran layanan secara internasional, tidak hanya AWS, memang harus menggunakan kartu bank yang support pembayaran internasional. Diantaranya yang banyak dijumpai yang berlogo VISA dan MasterCard. Tidak hanya credit card yang bisa, debit card juga bisa, asal ada support dari salah satu itu tadi.
.
Kalau kartu bank konvensional yang sudah ada logo VISA/MC nya tapi masih belum bisa transaksi online internasional biasanya memang harus aktivasi sendiri lewat CS bank.
.
Sejauh ini yang tidak perlu kontak2an lagi ke CS bisa pakai debit card Jenius, digibank by DBS, Bank Jago, Blu by BCA. Harusnya masih banyak lagi yang bisa tapi belum lihat ada yang lapor bisa pakai apa lagi.
---
>Untuk AWS pengalaman saya pake jenius debit card, pakai e-mail yang fresh contoh (mailto:***@gmail.com) Pastikan saldo cukup min 20.000 Kalau ada kendala pendebetan langsung kontak e-mail support aws nya.
---
>Kalau BCA debit lumayan praktis sih untuk activate/deactivate transaksi online dengan VISA/Mastercard. Bisa dikontrol via mBCA
---
>Kartu ATM Permata syariah bisa bang

Jika kamu punya pengalaman lain silahkan langsung saja tambahkan dikomentar.

**Reply by akriem** [reply]
saya pakai debit bank mandiri bisa bang

**Comment by za** [comment]
Terima kasih mas @rioastamal untuk ringkasan informasinya.

Berikutnya, perlu informasi bagaimana men-set limit kredit biar nanti tidak kaget saat membayar tagihan. Kasus: untuk penggunaan pribadi (bukan kantor). Hehehe 😁
